### PR TITLE
Adding more ad providers

### PR DESCRIPTION
**Pull request by shadyvb**

### CODE REVIEW

File: templates/ipinyou/py_buster.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+                } catch (error) {
+                    return '';
+                }
+            }
+            return '';
+        }
+        var bust = document.createElement('script'),
+            bustJS = getParameterByName('pybust');
+        if (bustJS !== '') {
+            bust.src = bustJS;
```
Comment (joshbetz) [inline templates/ipinyou/py_buster.html]: The domain in this script needs to be checked against a white list.

File: templates/predicta/predicta_bf.html [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+   }
+   else
+   {
+      setTimeout(function(){writeTag(fe, fep);} , 500);
+   } 
+}
+
+var lc = location.search.substring(1);
+var pp = getParam("p",";",lc);
+var dm = getParam("dm","&",lc);
```
Comment (joshbetz) [inline templates/predicta/predicta_bf.html]: This domain needs to be checked against a white list

File: templates/rockabox/rockabox_buster.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+                } catch (error) {
+                    return '';
+                }
+            }
+            return '';
+        }
+        var bust = document.createElement('script'),
+            bustJS = getParameterByName('rbbust');
+        if (bustJS !== '') {
+            bust.src = bustJS;
```
Comment (joshbetz) [inline templates/rockabox/rockabox_buster.html]: The domain needs to be checked against a white list

**Comment by nickdaugherty** [comment]
Ping @shadyvb on Josh's feedback - as-is, these files have some pretty big vulnerabilities that need to be fixed before they can be considered for inclusion in Adbusters.

**Comment by shadyvb** [comment]
@nickdaugherty these are all 3rd party scripts, and I'm not sure if they're still needed by the client ( I'm off the team now ). Probably safe to close as it's been quite a long time.

**Pull request closed without merging**